source/catalog.html

```diff
@@ -0,0 +1,259 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Каталог продукции Cat energy</title>
+</head>
+<body>
+<header class="page-header">
+  <div class="page-header__div">
+    <a class="cat-logo" href="index.html"><img class="cat-logo__image" src="#" alt="Логотип компании Кэт Энерджи."></a>
+    <a class="logo" href="index.html"><img class="logo__image" src="#" alt="Текстовый логотип Кэт Энерджи."></a>
+    <button class="page-header__button-close button-close" type="button" aria-label="Закрыть"></button>
+  </div>
+  <nav class="site-navigation">
+    <ul class="site-navigation__list site-nav-list">
+      <li class="site-nav-list__item"><a class="site-nav-list__link" href="catalog.html">Каталог продукции</a></li>
+      <li class="site-nav-list__item"><a class="site-nav-list__link" href="index.html">Главная</a></li>
+      <li class="site-nav-list__item"><a class="site-nav-list__link" href="form.html">Подбор программы</a></li>
+    </ul>
+  </nav>
+</header>
+<main class="page-main">
+  <h1 class="page-main__headline main-headline">Каталог продукции</h1>
+  <section class="product">
+    <h2 class="visually-hidden">Карточки товаров</h2>
+    <ul class="product__list product-list">
+      <li class="product-list__item">
+        <div class="product-div">
+          <div class="product-div__div">
+            <img class="product-div__product-image product-image" src="#" alt="Коробка с кормом Cat Energy Pro, вес 500 грамм, вкус - Курица.">
+            <div class="product-div__product-card-div">
+              <h3 class="product-div__product-name product-name"><a class="product-name__link" href="#">Cat Energy PRO
+                500 г</a></h3>
+              <dl class="product-div__product-description product-description">
+                <dt class="product-description__value">Объем</dt>
+                <dd class="product-description__quantity">500 г.</dd>
+                <dt class="product-description__value">Вкус</dt>
+                <dd class="product-description__quantity">Курица</dd>
+                <dt class="product-description__value">Цена</dt>
+                <dd class="product-description__quantity">700 Р.</dd>
+              </dl>
+            </div>
+          </div>
+          <a class="product-div__order-button order-button" href="#" role="button">Заказать</a>
+        </div>
+      </li>
+      <li class="product-list__item">
+        <div class="product-div">
+          <div class="product-div__div">
+            <img class="product-div__product-image product-image" src="#" alt="Коробка с кормом Cat Energy Pro, вес 1000 грамм, вкус - Курица.">
+            <div class="product-div__product-card-div">
+              <h3 class="product-div__product-name product-name"><a class="product-name__link" href="#">Cat Energy PRO
```

Comment:
Линка относится к диву всего товара
```suggestion
              <h3 class="product-div__product-name product-name"><a class="product-div__link" href="#">Cat Energy PRO
```